2-mandatory/6-journey-planner.js

```diff
@@ -123,8 +133,11 @@ function getLocationName() {}
 */
 function journeyPlanner(locations, transportMode) {
   // Implement the function body
+let accessibleLocations= locations.filter( location=> location.includes(transportMode))
```

Comment:
you should use the `isAccessibleByTransportMode()` function above here, but your solution is also fine